src/libaktualizr/uptane/uptane_network_test.cc

```diff
@@ -139,6 +140,33 @@ TEST(UptaneNetwork, DownloadFailure) {
   EXPECT_TRUE(result.first);
 }
 
+/*
+ * Output a log when connectivity is restored.
+ */
+TEST(UptaneNetwork, LogConnectivityRestored) {
+  testing::internal::CaptureStdout();
+  TemporaryDirectory temp_dir;
+  conf.storage.type = StorageType::kSqlite;
+  conf.storage.path = temp_dir.Path();
+  conf.provision.expiry_days = "noerrors";
+  conf.provision.primary_ecu_serial = "noerrors";
+  const std::string good_url = conf.provision.server;
+  auto http = std::make_shared<HttpClient>();
+  auto store = INvStorage::newStorage(conf.storage);
+
+  auto sota_client = std_::make_unique<SotaUptaneClient>(conf, store, http);
+  EXPECT_NO_THROW(sota_client->initialize());
+  result::UpdateCheck result = sota_client->fetchMeta();
```

Comment:
Got this failure `HTTP/1.0 501 Unsupported method ('PUT')`. Is `auto http = std::make_shared<HttpClient>();` a susceptive target?

Reply:
`HttpFake` seems to make no difference in local test.